packages/ruleset/src/functions/string-boundary.js

```diff
@@ -21,53 +21,71 @@ const bypassFormats = {
 };
 
 function stringBoundaryErrors(schema, path) {
-  // We're only interested in checking string properties.
```

Comment:
While developing and testing the new rule, I found a tweaks were required in the existing string-boundary rule.
I think an argument could be made to fold this rule's logic into the new property-attributes rule, but I didn't go that far :)